client/src/components/headerLogo.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
```

Comment:
I am wondering if this is necessary? Or can we do it without the client?

Reply:
I initially copied that from another file, so I wasn't sure if it was needed, neither. I just tested it by deleting "use client" in headerLogo.tsx, and I got the below error. So I believe we need it. 

Here is the Next.js doc about "use client" - [using client components in Next.js](https://nextjs.org/docs/app/building-your-application/rendering/client-components)

<img width="774" alt="createContext error" src="https://github.com/RubySpeeders/Drum-Roulette/assets/102322200/05cb5157-bad0-4540-85c4-8a929afa9e48">

Reply:
Do you know where the `createContext` is in this file? I was looking at the `CustomButton` component, which looks pretty similar to this one, which does not have `useClient` and wondering what the difference is.

Reply:
I did a search on every file but nothing came up for `createContext` .  I see that directory points to node modules, so it might be something related to that?